[2024-11-28T23:23:24.388-0600] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-28T23:23:24.391-0600] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: youtube_data_pipeline.extract_youtube_data manual__2024-11-29T05:23:22.559410+00:00 [queued]>
[2024-11-28T23:23:24.393-0600] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: youtube_data_pipeline.extract_youtube_data manual__2024-11-29T05:23:22.559410+00:00 [queued]>
[2024-11-28T23:23:24.393-0600] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-11-28T23:23:24.397-0600] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_youtube_data> on 2024-11-29 05:23:22.559410+00:00
[2024-11-28T23:23:24.399-0600] {standard_task_runner.py:72} INFO - Started process 41255 to run task
[2024-11-28T23:23:24.402-0600] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'youtube_data_pipeline', 'extract_youtube_data', 'manual__2024-11-29T05:23:22.559410+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/youtube_data_pipeline.py', '--cfg-path', '/var/folders/zs/sf3lbkjd0p58grc25nrggk1h0000gn/T/tmp1ceo1enm']
[2024-11-28T23:23:24.405-0600] {standard_task_runner.py:105} INFO - Job 20: Subtask extract_youtube_data
[2024-11-28T23:23:24.418-0600] {task_command.py:467} INFO - Running <TaskInstance: youtube_data_pipeline.extract_youtube_data manual__2024-11-29T05:23:22.559410+00:00 [running]> on host mac.attlocal.net
[2024-11-28T23:23:24.438-0600] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='youtube_data_pipeline' AIRFLOW_CTX_TASK_ID='extract_youtube_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-29T05:23:22.559410+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-29T05:23:22.559410+00:00'
[2024-11-28T23:23:24.439-0600] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-28T23:23:24.448-0600] {__init__.py:49} INFO - file_cache is only supported with oauth2client<4.0.0
[2024-11-28T23:23:24.505-0600] {logging_mixin.py:190} INFO - An error occurred: <HttpError 400 when requesting https://youtube.googleapis.com/youtube/v3/channels?part=snippet%2C+contentDetails%2C+statistics&id=UClFSU9_bUb4Rc6OYfTt5SPw%2CUCX6OQ3DkcsbYNE6H8uQQuVA%2CUCsooa4yRKGN_zEE8iknghZA%2CUCh9IfI45mmk59eDvSWtuuhQ%2CUCBJycsmduvYEL83R_U4JriQ%2CUC4rlAVgAK0SGk-yTfe48Qpw%2CUC0v-tlzsn0QZwJnkiaUSJVQ%2CUCzQUP1qoWDoEbmsQxvdjxgQ%2CUCoYATiiqNT7csPFqtWWfc5w%2CUC-lHJZR3Gqxm24_Vd_AJ5Yw%2CUCq-Fj5jknLsUf-MWSy4_brA&key=%2A%2A%2A%2A&alt=json returned "API key not valid. Please pass a valid API key.". Details: "[{'message': 'API key not valid. Please pass a valid API key.', 'domain': 'global', 'reason': 'badRequest'}]">
[2024-11-28T23:23:24.505-0600] {logging_mixin.py:190} INFO - Failed to retrieve data
[2024-11-28T23:23:24.542-0600] {logging_mixin.py:190} INFO - An error occurred: <HttpError 400 when requesting https://youtube.googleapis.com/youtube/v3/search?part=id%2C+snippet&channelId=UClFSU9_bUb4Rc6OYfTt5SPw&type=video&order=date&maxResults=10&key=%2A%2A%2A%2A&alt=json returned "API key not valid. Please pass a valid API key.". Details: "[{'message': 'API key not valid. Please pass a valid API key.', 'domain': 'global', 'reason': 'badRequest'}]">
[2024-11-28T23:23:24.571-0600] {logging_mixin.py:190} INFO - An error occurred: <HttpError 400 when requesting https://youtube.googleapis.com/youtube/v3/search?part=id%2C+snippet&channelId=UCX6OQ3DkcsbYNE6H8uQQuVA&type=video&order=date&maxResults=10&key=%2A%2A%2A%2A&alt=json returned "API key not valid. Please pass a valid API key.". Details: "[{'message': 'API key not valid. Please pass a valid API key.', 'domain': 'global', 'reason': 'badRequest'}]">
[2024-11-28T23:23:24.601-0600] {logging_mixin.py:190} INFO - An error occurred: <HttpError 400 when requesting https://youtube.googleapis.com/youtube/v3/search?part=id%2C+snippet&channelId=UCsooa4yRKGN_zEE8iknghZA&type=video&order=date&maxResults=10&key=%2A%2A%2A%2A&alt=json returned "API key not valid. Please pass a valid API key.". Details: "[{'message': 'API key not valid. Please pass a valid API key.', 'domain': 'global', 'reason': 'badRequest'}]">
[2024-11-28T23:23:24.636-0600] {logging_mixin.py:190} INFO - An error occurred: <HttpError 400 when requesting https://youtube.googleapis.com/youtube/v3/search?part=id%2C+snippet&channelId=UCh9IfI45mmk59eDvSWtuuhQ&type=video&order=date&maxResults=10&key=%2A%2A%2A%2A&alt=json returned "API key not valid. Please pass a valid API key.". Details: "[{'message': 'API key not valid. Please pass a valid API key.', 'domain': 'global', 'reason': 'badRequest'}]">
[2024-11-28T23:23:24.665-0600] {logging_mixin.py:190} INFO - An error occurred: <HttpError 400 when requesting https://youtube.googleapis.com/youtube/v3/search?part=id%2C+snippet&channelId=UCBJycsmduvYEL83R_U4JriQ&type=video&order=date&maxResults=10&key=%2A%2A%2A%2A&alt=json returned "API key not valid. Please pass a valid API key.". Details: "[{'message': 'API key not valid. Please pass a valid API key.', 'domain': 'global', 'reason': 'badRequest'}]">
[2024-11-28T23:23:24.698-0600] {logging_mixin.py:190} INFO - An error occurred: <HttpError 400 when requesting https://youtube.googleapis.com/youtube/v3/search?part=id%2C+snippet&channelId=UC4rlAVgAK0SGk-yTfe48Qpw&type=video&order=date&maxResults=10&key=%2A%2A%2A%2A&alt=json returned "API key not valid. Please pass a valid API key.". Details: "[{'message': 'API key not valid. Please pass a valid API key.', 'domain': 'global', 'reason': 'badRequest'}]">
[2024-11-28T23:23:24.739-0600] {logging_mixin.py:190} INFO - An error occurred: <HttpError 400 when requesting https://youtube.googleapis.com/youtube/v3/search?part=id%2C+snippet&channelId=UC0v-tlzsn0QZwJnkiaUSJVQ&type=video&order=date&maxResults=10&key=%2A%2A%2A%2A&alt=json returned "API key not valid. Please pass a valid API key.". Details: "[{'message': 'API key not valid. Please pass a valid API key.', 'domain': 'global', 'reason': 'badRequest'}]">
[2024-11-28T23:23:24.779-0600] {logging_mixin.py:190} INFO - An error occurred: <HttpError 400 when requesting https://youtube.googleapis.com/youtube/v3/search?part=id%2C+snippet&channelId=UCzQUP1qoWDoEbmsQxvdjxgQ&type=video&order=date&maxResults=10&key=%2A%2A%2A%2A&alt=json returned "API key not valid. Please pass a valid API key.". Details: "[{'message': 'API key not valid. Please pass a valid API key.', 'domain': 'global', 'reason': 'badRequest'}]">
[2024-11-28T23:23:24.810-0600] {logging_mixin.py:190} INFO - An error occurred: <HttpError 400 when requesting https://youtube.googleapis.com/youtube/v3/search?part=id%2C+snippet&channelId=UCoYATiiqNT7csPFqtWWfc5w&type=video&order=date&maxResults=10&key=%2A%2A%2A%2A&alt=json returned "API key not valid. Please pass a valid API key.". Details: "[{'message': 'API key not valid. Please pass a valid API key.', 'domain': 'global', 'reason': 'badRequest'}]">
[2024-11-28T23:23:24.848-0600] {logging_mixin.py:190} INFO - An error occurred: <HttpError 400 when requesting https://youtube.googleapis.com/youtube/v3/search?part=id%2C+snippet&channelId=UC-lHJZR3Gqxm24_Vd_AJ5Yw&type=video&order=date&maxResults=10&key=%2A%2A%2A%2A&alt=json returned "API key not valid. Please pass a valid API key.". Details: "[{'message': 'API key not valid. Please pass a valid API key.', 'domain': 'global', 'reason': 'badRequest'}]">
[2024-11-28T23:23:24.881-0600] {logging_mixin.py:190} INFO - An error occurred: <HttpError 400 when requesting https://youtube.googleapis.com/youtube/v3/search?part=id%2C+snippet&channelId=UCq-Fj5jknLsUf-MWSy4_brA&type=video&order=date&maxResults=10&key=%2A%2A%2A%2A&alt=json returned "API key not valid. Please pass a valid API key.". Details: "[{'message': 'API key not valid. Please pass a valid API key.', 'domain': 'global', 'reason': 'badRequest'}]">
[2024-11-28T23:23:24.886-0600] {logging_mixin.py:190} INFO - No comments were retrieved
[2024-11-28T23:23:24.887-0600] {logging_mixin.py:190} INFO - Failed to retrieve video categories
[2024-11-28T23:23:24.888-0600] {python.py:240} INFO - Done. Returned value was: None
[2024-11-28T23:23:24.897-0600] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-28T23:23:24.899-0600] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=youtube_data_pipeline, task_id=extract_youtube_data, run_id=manual__2024-11-29T05:23:22.559410+00:00, execution_date=20241129T052322, start_date=20241129T052324, end_date=20241129T052324
[2024-11-28T23:23:24.948-0600] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-28T23:23:24.959-0600] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-28T23:23:24.959-0600] {local_task_job_runner.py:245} INFO - ::endgroup::
